packages/vaex-ml/vaex/ml/transformations.py

```diff
@@ -431,17 +433,18 @@ def transform(self, df):
         :rtype: DataFrame
         '''
         copy = df.copy()
+        dtype = self.downcast_uint8 and 'uint8' or None
```

Comment:
We can put the check here right? And I'm never a fan of using `and` this way, I prefer `x if cond else y`
```suggestion
        downcast_uint8 = np.can_cast(self.one, np.uint8) and np.can_cast(self.zero, np.uint8)
        dtype = 'uint8' if downcast_uint8 else None
```